pkgs/development/libraries/agda/generics/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, mkDerivation, fetchFromGitHub, standard-library }:
+
+mkDerivation rec {
+  pname = "generics";
+  version = "rc";
+
+  src = fetchFromGitHub {
+    owner = "flupe";
+    repo = pname;
+    rev = "f52ef0f0afdf1503d32bcda7aab0f48ba875381f";
+    sha256 = "sha256-G5VGVZ5ehgPrWiv/CslRfZ4rU4CCijnrUO/R6xSygTg=";
+  };
+
+  buildInputs = [
+    standard-library
+  ];
+
+  everythingFile = "./README.agda";
+
+  meta = with lib; {
+    description =
+      "Library for datatype-generic programming in Agda";
+    homepage = src.meta.homepage;
+    license = licenses.publicDomain; # FIXME this isn't clear yet
```

Comment:
I could not find a license. See https://github.com/flupe/generics/issues/5

Reply:
See https://github.com/flupe/generics/issues/5#issuecomment-1006867315, it's probably going to be MIT, but let's wait until this is published.

Reply:
It's MIT.